### PR TITLE
Home, FNG, Slack, Contact pages

### PR DESCRIPTION
**Pull request by artberger**
Addresses:
* #8
* #10
* #11

### CODE REVIEW

File: content/contact/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Contact
+
+If you're on [Slack](/slack), you can directly message one of the [Leadership](/leadership) team.
+
+For website comments, see [GitHub](https://github.com/f3peakcity/f3peakcity.github.io).
+
+If you're new to F3, visiting from out of town, or just have a general question or comment, please fill out the following form.
+
+<iframe src="https://docs.google.com/forms/d/e/1FAIpQLSeVPJAdsFzyyyXDqA1WTic_62V291rdESRlMsvUJfKtt1zdWg/viewform?embedded=true" width="640" height="752" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```
Comment (artberger) [inline content/contact/index.md:9]: Embedded a Google Form to make sure it works...would like to transfer to the Peak City gmail and get it setup to forward there.

File: content/new-to-f3/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# New to F3?
+
+Everyone starts off as a "friendly new guy" (FNG) and then gets a slightly silly nickname after completing a workout.
```
Comment (artberger) [inline content/new-to-f3/index.md:3]: Content taken mostly from Carpex site, some from St. Louis, and I added a timeline at the bottom.

File: content/slack/index.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| #3rdF | Anything faith-related, such as praises, prayers, and other ways we can help each other out.|
+| #backblasts | A channel where backblasts, or reports for each workout of the day, are automatically added based on the Q filling out the **Paxmate** bot. You can also review the [Paxmate Dashboard](/paxmate-dashboard/) based on these reports. |
+| #cot-announcements | Announcements that you would like the pax to hear about in Circle of Trust at the end of each workout. These are usually upcoming events like a race or service project. Only admins can post to this channel. If you have an announcement, let someone on the [Leadership](/leadership/) team know.|
+| #mumblechatter | A catch-all channel for general questions, comments, observations of life, etc. Usually the most active channel on the site.|
+
+### Other channels of interest
+
+Channels are often used to share interests or coordinate activities around a certain type of event. As Peak City takes off, we are trying to take a "narrow and deep" approach to encourage activity (vs. a "wide and shallow" large number of channels with fewer participants).
```
Comment (artberger) [inline content/slack/index.md:28]: I thought keeping fewer channels would encourage denser posting in the beginning, so I tried to compress some of the popular ones into the following...not sure how many more we'd want. Maybe best to just remove and only describe default ones for now?

File: content/slack/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+### Default channels
+
+Everyone automatically is part of the following channels:
+
+| Channel | Description | 
+| --- | --- |
+| #1stF | Anything fitness-related, including "preblasts" or promotional info for who's leading each workout the night before.|
+| #2ndF | Anything fellowship-related, such as happy hours, spontaneous meetups to watch a game, or recommendations on things to do.|
+| #3rdF | Anything faith-related, such as praises, prayers, and other ways we can help each other out.|
+| #backblasts | A channel where backblasts, or reports for each workout of the day, are automatically added based on the Q filling out the **Paxmate** bot. You can also review the [Paxmate Dashboard](/paxmate-dashboard/) based on these reports. |
```
Comment (artberger) [inline content/slack/index.md:22]: Thoughts on a separate channel for backblasts to reduce Slack clutter? It was one of the suggestions at an Exchange. I personally don't mind them in 1stF.

File: content/slack/index.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+| Channel | Description | 
+| --- | --- |
+| #buy-sell-trade | Things you're selling or looking for, or deals that you found that other pax might be interested in, kind of like an F3 Marketplace. |
+| #diy-home-ec | Anything related to DIY projects, house projects, tools of the trade, recommendations for local service providers, and so on. |
+| #business-finance-career | Anything related to money, personal finances, career advice, jobs, professional development, and so on. |
+| #healthy-eats | Anything cooking, nutrition, diet-related. |
+
+## Moderation
+
+As with everything in F3, you are "Freed to Lead," and can post mostly whatever you like, or even help the [Commz team](/leadership/) moderate.
```
Comment (artberger) [inline content/slack/index.md:41]: Any other moderation guidelines we'd want?

File: content/_index.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+## Get a free workout
+Join us for 20+ free workouts right here in the "Peak of Good Living" in Apex, NC and the surrounding areas of Friendship, New Hill, and Jordan Lake.
+
+The workouts are men of all fitness levels and goals, and include running; cardio with total-body and body-weight exercises; limited run and strength-building with dumbells, kettleballs, and other toys; walking; rucking; and even yoga and stretching.
+
+{{< column >}}
+
+{{< figure src="/img/workout-hill.jpeg" title="Example workout running up a hill" >}}
+
+{{< button relref="/schedule" >}}Find workouts{{< /button >}}
```
Comment (artberger) [inline content/_index.md:13]: Added buttons to the theme.

File: config.toml [hunk 7]
```diff
@@ -99,11 +99,17 @@ unsafe = true
     name = "Schedule"
     weight = 1
 
+[[menu.main]]
```
[the remark this answers is not in the document]
Comment (mikeyrcamp) [reply config.toml:102]: /lgtm

File: content/slack/index.md (hunk 4 above)
Comment (mikeyrcamp) [reply content/slack/index.md:22]: It ends up being a great comment thread after each workout. Moving them will slide things around a bit more. It also with the mentions engages folks, but yeah not really sure. I'm adverse to changing too much, but we can certainly think about it.

File: content/slack/index.md (hunk 5 above)
Comment (mikeyrcamp) [reply content/slack/index.md:41]: I think these are good. Only other I think of is call out explicitly spamming @channel and @here.

File: content/slack/index.md (hunk 3 above)
Comment (mikeyrcamp) [reply content/slack/index.md:28]: I think the default ones can have a description in their header/description (in slack) so having them here is fine, but duplicated. However, this is a far more complete document giving an overview so I believe it's fine. We'll have to update the links.

File: content/new-to-f3/index.md (hunk 2 above)
Comment (mikeyrcamp) [reply content/new-to-f3/index.md:3]: 👍

File: content/slack/index.md (hunk 4 above)
Comment (artberger) [reply content/slack/index.md:22]: Will remove

File: content/contact/index.md (hunk 1 above)
Comment (mikeyrcamp) [reply content/contact/index.md]: amended your commit and switched the form link with one hosted by peak city gmail. Thanks!

**Review by mikeyrcamp — APPROVED** [review]

**Pull request merged**